production/historical_scrape/historical_scrapers/historical_texas_deaths.R

```diff
@@ -0,0 +1,105 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+historical_texas_deaths_pull <- function(x, date = NULL, file = NULL){
+    z <- "1qnea3D6S18H2tKz0Q1STnJ47qkXxNE8u-axPA1p81Zc" %>%
+        googlesheets4::read_sheet(
+            sheet = "Sheet1",
+            )
+}
+
+historical_texas_deaths_restruct <- function(z){
+    # check against current scrape date
+    tx_dat <- read_mpap_data(all_dates = TRUE) %>%
+        filter(State == "Texas")
+    
+    out <- z %>%
+        select(Date = `Date (YYYY-MM-DD)`,
+               New.Residents.Deaths = Residents.Deaths, 
+               Name) %>%
+        filter(!is.na(Date)) %>%
+        mutate(Date = lubridate::ymd(Date)) %>%
+        group_by(Name) %>% 
+        arrange(Date) %>%
+        mutate(Residents.Deaths = cumsum(New.Residents.Deaths)) %>%
+        ungroup() %>%
+        select(-New.Residents.Deaths)
+    
+    ## compare our data to marshall project, our historical data 
+    # cbb <- out %>%
+    #     group_by(month_yr = glue('{month(Date)}_{year(Date)}'), Name) %>% 
+    #     summarise(Residents.Deaths_max_fac_month = max(Residents.Deaths)) %>% # get max cumulative count from each facility in each month 
+    #     group_by(month_yr) %>% 
+    #     ## need to arrange by correct date order and take the cumulative sum 
+    #     summarize(Residents.Deaths_cbb_manual = sum(Residents.Deaths_max_fac_month),
+    #               n = n()) 
+    
+    # mpap <- tx_dat %>%
+    #     group_by(month_yr = glue('{month(Date)}_{year(Date)}')) %>% 
+    #     summarise(Residents.Deaths_mpap = max(Residents.Deaths)) 
+    
+    # historical_deaths <- read_csv("~/Desktop/historical_tx_deaths.csv")
+    
+    ## some strange things going on here
+    # compare <- cbb %>%
+    #     left_join(mpap, by = "month_yr") %>%
+    #     # left_join(historical_deaths, by = "month_yr") %>%
+    #     relocate(month_yr, "Residents.Deaths_mpap", starts_with("Residents.Deaths_cbb"))
+}
+
+historical_texas_deaths_extract <- function(x){
+    x
```

Comment:
can we add a stop_defunct scraper call here so we dont accidentally write anything?